packages/gafl-webapp-service/src/pages/renewals/renewal-inactive/__tests__/route.spec.js

```diff
@@ -1,45 +1,230 @@
+      }))
+
+      const { ...titleAndBodyMessage } = await getData({
+        cache: () => ({
+          helpers: {
+            status: {
+              getCurrentPermission: mockStatusCacheGet
+            }
+          }
+        }),
+        i18n: {
+          getCatalog: jest.fn(() => englishTranslations)
+        },
+        locale: ''
+      })
+
+      const { bodyMessage } = getTitleAndBodyMessage(englishTranslations, reason, referenceNumber, formatSymbol)
+
+      expect(titleAndBodyMessage.bodyMessage).toEqual(bodyMessage)
+    })
+
+    it('endDate is passed to moment', async () => {
+      const endDate = Symbol('endDate')
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate },
+        referenceNumber: 'ABC123'
+      }))
+      moment.mockImplementation(getMomentMockImpl())
+      await getData(mockRequest)
+      expect(moment).toHaveBeenCalledWith(endDate, expect.any(String), expect.any(String))
+    })
+
+    it('cacheDateFormat being what is expected', async () => {
       mockStatusCacheGet.mockImplementationOnce(() => ({
         authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate: '2020-12-13T23:59:59Z' },
         referenceNumber: 'ABC123'
       }))
-      const result = await getData(mockRequest)
-      expect(result.bodyMessage).toBe('The licence ending in ABC123 does not expire until 13 December 2020')
+      moment.mockImplementation(getMomentMockImpl())
+      await getData(mockRequest)
+      expect(moment).toHaveBeenCalledWith(expect.any(String), cacheDateFormat, expect.any(String))
+    })
+
+    it('locale is set on moment, to whatever the request.locale is', async () => {
+      const language = Symbol('endDate')
+      const mockRequest = {
+        cache: () => ({
+          helpers: {
+            status: {
+              getCurrentPermission: mockStatusCacheGet
+            }
+          }
+        }),
+        i18n: {
+          getCatalog: jest.fn(() => englishTranslations)
+        },
+        locale: language
+      }
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate: '2020-12-13T23:59:59Z' },
+        referenceNumber: 'ABC123'
+      }))
+      const format = jest.fn()
+      moment.mockImplementation(getMomentMockImpl({ format }))
+      await getData(mockRequest)
+      expect(moment).toHaveBeenCalledWith(expect.any(String), expect.any(String), language)
+    })
+
+    it('dateDisplayFormat is passed to format', async () => {
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate: '2020-12-13T23:59:59Z' },
+        referenceNumber: 'ABC123'
+      }))
+      const format = jest.fn()
+      moment.mockImplementation(getMomentMockImpl({ format }))
+      await getData(mockRequest)
+      expect(format).toHaveBeenCalledWith(dateDisplayFormat)
```

Comment:
Again, you're mocking it now, but you're not using a Symbol. Fine for most tests, just override it as a Symbol for this one.